layers/+lang/typescript/packages.el

```diff
@@ -42,8 +42,19 @@
   (spacemacs/enable-flycheck 'typescript-tsx-mode)
   (with-eval-after-load 'tide
     (with-eval-after-load 'flycheck
-      (flycheck-add-mode 'typescript-tide 'typescript-tsx-mode)
-      (flycheck-add-mode 'typescript-tslint 'typescript-tsx-mode))))
+      (if (eq typescript-linter `tslint)
```

Comment:
Would probably be better to use `cond` and check for explicit `'eslint` and `'tslint` values instead (and complain if it's anything else)

Reply:
Added the cond check - just writes to the message buffer, not sure if there is a better practice way to do it.

Reply:
I think this is ok so far, of what I have seen in other layers i.e. ruby if something non matching is given an message is written to the log and not an error is raised.